src/libos/src/util/sync/rw_lock/inner.rs

```diff
@@ -137,62 +173,70 @@ impl RwLockInner {
         }
     }
 
-    pub fn try_read(&self) -> Result<()> {
+    pub(super) fn try_read(&self) -> Result<()> {
         loop {
-            let val: i32 = self.rw_lock.load(Ordering::SeqCst);
-            let cnt: i32 = val & 0x7FFF_FFFF;
-            if cnt == 0x7FFF_FFFF {
-                return_errno!(EBUSY, "a writer is holding the lock");
-            }
-            if cnt == 0x7FFF_FFFE {
-                return_errno!(EAGAIN, "the maximum number of read locks has been exceeded");
+            let current_status = self.status();
```

Comment:
`let current_status = self.status();`
should be 
`let current_status = self.status().get_locker();`

Reply:
Done. Thanks for your review.